src/arrayMethodSort.js

```diff
@@ -5,8 +5,23 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let callBackFn = compareFunction;
```

Comment:
```suggestion
    const callBackFn = callBackFn ? compareFunction : defaultCallBackFn 
```
And then you can use this variant (alternative)